add_service.rb

```diff
@@ -0,0 +1,22 @@
+def add_service(option)
+  case option
+  when 3
+    add_book
+  when 4
+    add_music_album
+  when 5
+    add_game
+  end
```

Comment:
- [ ] As implemented in your `select_option` method, kindly update the option cases here to:
 - 7: add_book
 - 8: add_music_album
 - 9: add_game